src/config.ts

```diff
@@ -25,6 +25,10 @@ const config = {
         PROD: 'https://op-build-prod.azurewebsites.net',
         PPE: 'https://op-build-sandbox2.azurewebsites.net'
     },
+    AIKey: {

```

Comment:
where are the values from?

Reply:
when you create a Application Insight instance, you will one key

---
In reply to: [376910284](https://github.com/docascode/vscode-docs-build/pull/35#discussion_r376910284) [](ancestors = 376910284)